Tasks/app-store-release/app-store-release.ts

```diff
@@ -43,6 +43,27 @@ function findIpa(ipaPath: string): string {
     return paths[0];
 }
 
+async function isFastlaneInstalled(): Promise<boolean> {
+    tl.debug('Checking for previous fastlane version first...');
+    let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
+    gemRunner.arg(['list', 'fastlane']);
+    gemRunner.arg(['-i']);
+    let fastlaneListOutput: string;
+
+    gemRunner.on('stdout', (data) => {
+        fastlaneListOutput = data.toString().trim();
+    });
+
+    await gemRunner.exec();
+
+    if (fastlaneListOutput === 'true') {
+        // fastlane is installed
+        return Promise.resolve(true);
```

Comment:
you can just `return true;` and it will get "lifted" to a Promise

Reply:
but if you use `execSync` this function won't need to be async anymore

Reply:
Thanks for sharing, didn't know that but I prefer to be explicit. I am removing this and switching to execSync anyway.